product-payment.html

```diff
@@ -0,0 +1,290 @@
+            <div class="product-page__extend">
+              <a href="" class="product-page__button">
+                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M12 22C12 22 20 18 20 12V4L12 2L4 4V12C4 18 12 22 12 22Z" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                </svg>
+                <p class="product-page__warranty"> Extender garantia <svg width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M1 1L7 7L13 1" stroke="#EF2A82" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                  </svg>
+                </p>
+              </a>
+              <div class="product-page__requires">
+                <p class="product-page__penalties">¿Requieres inspección? <a href="">
+                    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                      <path fill-rule="evenodd" clip-rule="evenodd" d="M8.00016 14.6667C11.6821 14.6667 14.6668 11.6819 14.6668 8.00004C14.6668 4.31814 11.6821 1.33337 8.00016 1.33337C4.31826 1.33337 1.3335 4.31814 1.3335 8.00004C1.3335 11.6819 4.31826 14.6667 8.00016 14.6667Z" stroke="#EF2A82" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                      <path d="M8 10.6667V8" stroke="#EF2A82" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                      <circle cx="8.00016" cy="5.33329" r="0.666667" fill="#EF2A82" />
+                    </svg>
+                  </a>
+                </p>
+                <div class="product-page__input">
+                  <input class="product-page__thanks" type="text" placeholder="No, gracias">
+                  <a class="product-page__svg" href="">
+                    <svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                      <path d="M4.10938 6L8.10938 10L12.1094 6" stroke="#363535" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                    </svg>
+                  </a>
+                </div>
+              </div>
+            </div>
+            <a href="#" class="product-page__learn-more">Conocer más <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M4 12L12 4" stroke="#EF2A82" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M6 4H12V10" stroke="#EF2A82" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+              </svg>
+            </a>
+            <div class="product-page__bords"></div>
+            <p class="product-page__method">Metodo de entrega</p>
+            <div class="product-page__method-page">
+              <a href="" class="product-page__ofer">
```

Comment:
Este par puede ser un radiobutton estilizado, ya que vas a escoger un tipo de entrega entre dos tipos
```suggestion
              <label class="product-page__ofer">
                <input type="radio" name="entrega" class="product-page__delivery-type">
```